protocol/consistencychecks_test.go

```diff
@@ -3,29 +3,28 @@ package protocol
 import "testing"
 
 var (
-	key = []byte("key")
+	alice = "alice"
+	bob   = "bob"
+	key   = []byte("key")
 )
 
-func doRequestAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
-	requestType int, name string) error {
-	switch requestType {
-	case RegistrationType:
-		request := &RegistrationRequest{
-			Username: name,
-			Key:      key,
-		}
-		res, _ := d.Register(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case KeyLookupType:
-		request := &KeyLookupRequest{
-			Username: name,
-		}
-		res, _ := d.KeyLookup(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case MonitoringType:
-	case KeyLookupInEpochType:
+func registerAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
```

Comment:
I'm not saying you're wrong; just that the tests would be more understandable to me that way.  There's nothing in `registerAndVerify(d, cc, alice, key)` returning `CheckPassed` that indicates to me that the `error name existed`.  The registration could have equally well succeeded.  We know it couldn't, because of the tests elsewhere, as you said, but that just isn't clear when trying to read this linearly.